core/build.sh

```diff
@@ -139,24 +144,21 @@ do_action clean
 
 do_action configure
 
-# After we have built the necessary internal dependencies, then we can go
-# ahead and build a fat library for external consumption
-if builder_start_action configure:mac; then
-  mkdir -p "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION"
-  builder_finish_action success configure:mac
-fi
+# After we have built the necessary internal dependencies, then we can go ahead
+# and build a fat library for external consumption for mac, and no-op for win
+builder_run_action configure:mac   mkdir -p "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION"
+builder_run_action configure:win   mkdir -p "$KEYMAN_ROOT/core/build/win/$BUILDER_CONFIGURATION"
 
 # -------------------------------------------------------------------------------
 
 do_action build
 
-if builder_start_action build:mac; then
-  lipo -create \
-    "$KEYMAN_ROOT/core/build/mac-x86_64/$BUILDER_CONFIGURATION/src/libkeymancore.a" \
-    "$KEYMAN_ROOT/core/build/mac-arm64/$BUILDER_CONFIGURATION/src/libkeymancore.a" \
-    -output "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION/libkeymancore.a"
-  builder_finish_action success build:mac
-fi
+builder_run_action build:mac lipo -create \
+  "$KEYMAN_ROOT/core/build/mac-x86_64/$BUILDER_CONFIGURATION/src/libkeymancore.a" \
+  "$KEYMAN_ROOT/core/build/mac-arm64/$BUILDER_CONFIGURATION/src/libkeymancore.a" \
+  -output "$KEYMAN_ROOT/core/build/mac/$BUILDER_CONFIGURATION/libkeymancore.a"
+
+builder_run_action build:win touch "$KEYMAN_ROOT/core/build/win/$BUILDER_CONFIGURATION/BUILT"
```

Comment:
Does /BUILT ever get cleaned?

Reply:
Good question -- not at this point. Might be worth a future PR to do clean for both :mac and :win.